__tests__/app.test.js

```diff
@@ -5,98 +5,139 @@ const seed = require('../db/seeds/seed.js')
+    return request(app)
+      .get('/api/articles')
+      .expect(200)
+      .then(({ body: { articles } }) => {
+        expect(articles).toHaveLength(12);
+        articles.forEach((article) => {
+          expect(article).toMatchObject({
+            author: expect.any(String),
+            title: expect.any(String),
+            article_id: expect.any(Number),
+            topic: expect.any(String),
+            created_at: expect.any(String),
+            votes: expect.any(Number),
+            comment_count: expect.any(String),
+          })
         });
-    })
+      });
+  })
 })
 
 describe('3. GET /api/articles/:article_id', () => {
-    test('status:200, responds with a single matching article', () => {
-      const articleId = 3;
-      return request(app)
-        .get(`/api/articles/${articleId}`)
-        .expect(200)
-        .then((response) => {
-            const article = response.body.article
-          expect(article).toEqual({
-            author: "icellusedkars",
-            title: 'Eight pug gifs that remind me of mitch',
-            article_id: articleId,
-            body: "some gifs",
-            topic: 'mitch',
-            created_at: "2020-11-03T09:12:00.000Z",
-            votes: 0,
-          });
+  test('status:200, responds with a single matching article', () => {
+    const articleId = 3;
+    return request(app)
+      .get(`/api/articles/${articleId}`)
+      .expect(200)
+      .then((response) => {
+        const article = response.body.article
+        expect(article).toEqual({
+          author: "icellusedkars",
+          title: 'Eight pug gifs that remind me of mitch',
+          article_id: articleId,
+          body: "some gifs",
+          topic: 'mitch',
+          created_at: "2020-11-03T09:12:00.000Z",
+          votes: 0,
         });
-    });
-    test('400: invalid article_id', () => {
-        return request(app)
-        .get('/api/articles/bananas')
-        .expect(400)
-        .then((response) => {
-            const msg = response.body.msg;
-            expect(msg).toBe('Bad Request')
-        })
-    })
+      });
+  });
+  test('400: invalid article_id', () => {
+    return request(app)
+      .get('/api/articles/bananas')
+      .expect(400)
+      .then((response) => {
+        const msg = response.body.msg;
+        expect(msg).toBe('Bad Request')
+      })
+  })
+
+  test('404: valid article_id but doesnt exist', () => {
+    return request(app)
+      .get('/api/articles/30')
+      .expect(404)
+      .then((response) => {
+        const msg = response.body.msg;
+        expect(msg).toBe('Not Found')
+      })
+  })
+
+});
 
-    test('404: valid article_id but doesnt exist', () => {
-        return request(app)
-        .get('/api/articles/30')
-        .expect(404)
-        .then((response) => {
-            const msg = response.body.msg;
-            expect(msg).toBe('Not Found')
+describe('4. GET /api/articles/:article_id/comments', () => {
```

Comment:
Please could you add a test for an empty array if the article exists but doesn't have comments